[2023-06-14T13:39:48.780+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.insert_into_table manual__2023-06-14T13:39:45.989112+00:00 [queued]>
[2023-06-14T13:39:48.786+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.insert_into_table manual__2023-06-14T13:39:45.989112+00:00 [queued]>
[2023-06-14T13:39:48.787+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T13:39:48.787+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-06-14T13:39:48.787+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T13:39:48.795+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-06-14 13:39:45.989112+00:00
[2023-06-14T13:39:48.799+0000] {standard_task_runner.py:55} INFO - Started process 1040 to run task
[2023-06-14T13:39:48.802+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_postgres', 'insert_into_table', 'manual__2023-06-14T13:39:45.989112+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/postgress_db.py', '--cfg-path', '/tmp/tmp84slh_3z']
[2023-06-14T13:39:48.803+0000] {standard_task_runner.py:83} INFO - Job 29: Subtask insert_into_table
[2023-06-14T13:39:48.850+0000] {task_command.py:388} INFO - Running <TaskInstance: database_postgres.insert_into_table manual__2023-06-14T13:39:45.989112+00:00 [running]> on host 649eec7cdf92
[2023-06-14T13:39:48.896+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_postgres
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-14T13:39:45.989112+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-14T13:39:45.989112+00:00
[2023-06-14T13:39:48.898+0000] {sql.py:254} INFO - Executing: INSERT INTO my_table (id,"current_time") VALUES (1,CURRENT_TIMESTAMP);
[2023-06-14T13:39:48.905+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T13:39:49.005+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T13:39:49.009+0000] {sql.py:375} INFO - Running statement: INSERT INTO my_table (id,"current_time") VALUES (1,CURRENT_TIMESTAMP);, parameters: None
[2023-06-14T13:39:49.011+0000] {sql.py:384} INFO - Rows affected: 1
[2023-06-14T13:39:49.019+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_postgres, task_id=insert_into_table, execution_date=20230614T133945, start_date=20230614T133948, end_date=20230614T133949
[2023-06-14T13:39:49.058+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-06-14T13:39:49.075+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
